test_launch_ros/test/test_launch_ros/xml_frontend/node.xml

```diff
@@ -0,0 +1,12 @@
+<launch>
+    <node package="demo_nodes_py" executable="talker_qos" output="screen" name="my_node" namespace="my_ns" args="--number_of_cycles 1">
```

Comment:
@ivanpauno hmm I'm not sure if we should be using demo nodes for testing of core packages. Is that the case already? Otherwise, consider rolling out a dummy one just for this purpose.

Reply:
We're already doing that.
https://github.com/ros2/launch_ros/blob/67d0c886e0245b5b4d7bd0e58564f07bdc13075f/test_launch_ros/package.xml#L1-L23
https://github.com/ros2/launch_ros/blob/67d0c886e0245b5b4d7bd0e58564f07bdc13075f/test_launch_ros/test/test_launch_ros/actions/test_node.py#L43-L52

Reply:
Hmm, ok. I wonder still. But OK to keep it as is now.